apps/extension/src/wallet-services.ts

```diff
@@ -86,7 +86,11 @@ const attachServiceControlListener = ({
       case ServicesMessage.ClearCache:
         void (async () => {
           blockProcessor.stop('clearCache');
-          await Promise.allSettled([localExtStorage.remove('params'), indexedDb.clear()]);
+          await Promise.allSettled([
+            localExtStorage.remove('params'),
+            indexedDb.clear(),
+            chrome.storage.local.remove('fullSyncHeight'),
```

Comment:
Ah, unfortunately using the chrome storage api directly is not typesafe and could cause bugs with old Prax versions that have not undergone db migrations. Can you follow up with a PR to do:

```diff
- chrome.storage.local.remove('fullSyncHeight'),
+ localExtStorage.remove('fullSyncHeight'),
```